packages/core/entity-list/src/modules/list/sagas.js

```diff
@@ -417,6 +417,10 @@ export function* remoteEvent(action) {
     }
   }
 
+  if (event.type === 'trigger-action-event') {
```

Comment:
other remote events have naming schema from `<name>-<verb>-event`. Should we maybe rename this one to `action-trigger-event` instead?